apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -93,6 +109,41 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
     });
   }
 
+  /**
+   * Helper to detect the firmware (major and minor) version of the given playground
+   * @param playground
+   */
+  detectFirmwareVersion(playground) {
```

Comment:
Never called? I'd love to `this.detectFirmwareVersion(playground);` in the "on ready" handler here, to exercise this code path:

https://github.com/code-dot-org/code-dot-org/blob/87c19d3aa5959f96390fca5b028fc1de3d2ea72f/apps/src/lib/kits/maker/CircuitPlaygroundBoard.js#L85-L90